### PR TITLE
Application

### PR DESCRIPTION
**Pull request by Alejandra230701**
## What 
This pull request removes the environment variable “Url” and introduces the new environment variable “Application” which retrieves the URL based on the application name.
## How
-	Removed environment variable “Url”
-	Added environment variable “Application”
-	Added “getApplicationUrl” method to retrieve the URL corresponding to the application name.
## Tests added 
 
image

1.	Verify IllegalArgumentException is thrown when the application name is null.
 
image

2.	Verify IllegalArgumentexception is thrown when the application name is incorrect
Application = amazon
 
image

3.	Verify the correct URL is returned when the application name is correct
Application = mrc
 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,30 +51,64 @@ Navigate to the directory where you cloned the repository and select it.
...
+|---------------|--------------------------------------|
+| Accessibility | true                                 |
+| Platform      | Web                                  |
+| Application   | xxxx                                 |
+| Browser       | xxxx (default: chrome,can be null)   |
+| Resolution    | xxxx (default:1024x768, can be null) |
 
+#### Notes
+
+- The 'Browser' variable can take the value of 'null', in which case the default browser used for testing is 'chrome'.
```
Comment (raunelgarcia) [inline README.md]: toda esta explicacion es mejor incluirla al principio donde explicas cada una, asi no estamos repitiendo mas de lo mismo

File: README.md [hunk 2]
```diff
@@ -104,18 +138,18 @@ If you want to generate reports with Allure, the framework is ready for it. Befo
 
 Install Allure in the project using the command `npm install --save-dev allure-commandline`
 
-Once it's installed, you can use Allure commands from the project directory. When the test is executed, the results will be saved in target/allure-results
-
-To generate HTML reports, you'll need to execute the command `npx allure-commandline generate target/allure-results`
+Once it's installed and when the test is executed, the results will be saved in target/allure-results. A browser window will open automatically, displaying the Allure report.
```
Comment (raunelgarcia) [inline README.md:133]: agregar tambien el path donde se reportaran los resultados de accesibilidad si esta activada

File: README.md (hunk 1 above)
Comment (Alejandra230701) [reply README.md]: Listo, he eliminado notas y lo he puesto todo en las descriptions

File: README.md (hunk 2 above)
Comment (Alejandra230701) [reply README.md:133]: vale, he agregado una nueva sección para accessibility

**Review by raunelgarcia — APPROVED** [review]

**Pull request merged**